python/ql/src/experimental/dataflow/internal/DataFlowPublic.qll

```diff
@@ -69,11 +80,13 @@ class EssaNode extends Node, TEssaNode {
 }
 
 class CfgNode extends Node, TCfgNode {
-  ControlFlowNode node;
+  DataFlowCfgNode node;
 
   CfgNode() { this = TCfgNode(node) }
 
-  ControlFlowNode getNode() { result = node }
+  DataFlowCfgNode getNode() { result = node }
```

Comment:
```suggestion
  ControlFlowNode getNode() { result = node }
```

Reply:
Not as sure about this one, but let us try it and see how it works :)